rust-runtime/aws-smithy-runtime-api/src/client/auth.rs

```diff
@@ -120,6 +121,12 @@ impl AuthSchemeOptionResolver for SharedAuthSchemeOptionResolver {
     }
 }
 
+impl_shared_conversions!(
+    convert SharedAuthSchemeOptionResolver
+    from AuthSchemeOptionResolver
+    using SharedAuthSchemeOptionResolver::new
+);
```

Comment:
I'd probably consider adding `:` in the macro to remove any parsing hazard but this seems to be working fine

Reply:
```
convert: SharedAuthSchemeOptionResolver
from: Auth....
```